### PR TITLE
add page titles to every page + copy to clipboard on treasury

### PR DESCRIPTION
**Pull request by herbig**
## Description

- Adds page titles to every page, according to the designs.
- Adds the "copy to clipboard" component to the Treasury page

## Issue / Notion doc (if applicable)
Closes https://github.com/decent-dao/fractal-interface/issues/879

## Testing

- [x] Every page with a PageHeader is impacted, please ensure no adverse impacts do to this change.
- [x] Please also check that the "copy to clipboard" component on the DAO homepage also has no UI impacts, as this uses the same component as the Treasury

## Screenshots (if applicable)
Screenshot 2023-04-27 at 4 29 37 PM

Screenshot 2023-04-28 at 1 59 30 PM

### CODE REVIEW

File: app/daos/[daoAddress]/hierarchy/page.tsx [hunk 1]
```diff
@@ -30,9 +31,13 @@ export default function HierarchyPage() {
...
       <Box>
         <PageHeader
+          title={t('headerTitle', {
+            daoName: daoName,
+            subject: t('nodes'),
+          })}
           breadcrumbs={[
             {
-              title: t('nodes', { ns: 'breadcrumbs' }),
+              terminus: t('nodes'),
```
Comment (herbig) [inline app/daos/[daoAddress]/hierarchy/page.tsx:40]: @Da-Colon I changed the term for the breadcrumbs to "terminus", so it didn't conflict with / be confused for the page title

File: app/daos/[daoAddress]/proposals/new/page.tsx [hunk 2]
```diff
@@ -128,7 +122,7 @@ export default function ProposalCreatePage() {
                     >
                       <Box
                         rounded="lg"
-                        p="5rem"
+                        p="1rem"
```
Comment (herbig) [inline app/daos/[daoAddress]/proposals/new/page.tsx:125]: this was a bug in the proposal creator that just had too much padding so fixed that here as well

File: src/components/ui/page/Header/PageHeader.tsx [hunk 3]
```diff
@@ -7,12 +7,14 @@ import {
...
   IconProps,
   Spacer,
+  Text,
 } from '@chakra-ui/react';
 import { ReactNode, useEffect, useState } from 'react';
 import { DAO_ROUTES } from '../../../../constants/routes';
 import { useFractal } from '../../../../providers/App/AppProvider';
 import Breadcrumbs, { Crumb } from './Breadcrumbs';
 interface IPageHeader {
+  title?: string;
```
Comment (herbig) [inline src/components/ui/page/Header/PageHeader.tsx:17]: title is optional, there's a few places we use PageHeader without a title, such as the Proposals list page

File: src/i18n/locales/en/proposal.json [hunk 4]
```diff
@@ -10,7 +11,6 @@
     "helperParameters": "Parameters used to call the function (comma separated)",
     "labelEthValue": "ETH Value",
     "helperEthValue": "Amount of ETH to send with the transaction (from your treasury)",
-    "pageTitle": "{{daoName}} Proposals",
```
Comment (herbig) [inline src/i18n/locales/en/proposal.json:13]: unused string removed

File: src/components/ui/cards/DAOInfoCard.tsx [hunk 5]
```diff
@@ -53,12 +46,10 @@ export function DAOInfoCard({
...
   } = useFractal();
   const { favoritesList, toggleFavorite } = useAccountFavorites();
-  const copyToClipboard = useCopyText();
   const isCurrentDAO = safeAddress === daoAddress;
   const { daoRegistryName } = useDAOName({
     address: safeAddress && !isCurrentDAO ? safeAddress : undefined,
   });
-  const { accountSubstring } = useDisplayName(safeAddress);
 
   // @todo add viewable conditions
```
Comment (Da-Colon) [inline src/components/ui/cards/DAOInfoCard.tsx:54]: this todo can be removed yeah?

File: src/components/ui/links/AddressCopier.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <Flex
+      alignItems="center"
+      onClick={() => copyToClipboard(address)}
+      gap="0.5rem"
+      cursor="pointer"
+      w="fit-content"
+      {...rest}
+    >
+      <EtherscanLinkAddress address={address}>
```
Comment (Da-Colon) [inline src/components/ui/links/AddressCopier.tsx:28]: I believe this component already has the copy hook

**Review by Da-Colon — COMMENTED** [review]
Looks good, a few comments

File: src/components/ui/links/AddressCopier.tsx (hunk 6 above)
Comment (herbig) [reply src/components/ui/links/AddressCopier.tsx:28]: oh shoot you're right ha

Comment (herbig) [reply src/components/ui/links/AddressCopier.tsx:28]: @Da-Colon I did a bunch of refactoring, and took it out of the `EtherscanLinkAddress`, I think that should just be a plain link, and let other components add functionality, the only use of that was in modules, which will have a proper design in settings soon anyway

**Review by Da-Colon — APPROVED** [review]
LGTM! 🚀

**Pull request merged**